doc/api/crypto.md

```diff
@@ -972,6 +1130,9 @@ The `key` is the raw key used by the `algorithm` and `iv` is an
 [buffers][`Buffer`].
 
 ### crypto.createDiffieHellman(prime[, prime_encoding][, generator][, generator_encoding])
```

Comment:
This signature was added in v0.11.12. Maybe it makes sense to use this version instead of v0.5.0. See below for the original signature.